src/components/swap/TradePrice.tsx

```diff
@@ -33,6 +33,11 @@ export default function TradePrice({ price, showInverted, setShowInverted }: Tra
   const theme = useContext(ThemeContext)
 
   const usdcPrice = useUSDCPrice(showInverted ? price.baseCurrency : price.quoteCurrency)
+  /*
+   * calculate needed amount of decimal prices, for prices between 0.95-1.05 use 4 decimal places
+   */
+  const p = Number(usdcPrice?.toFixed(3))
```

Comment:
Nit: can you just use `toFixed()` here with the default value?

Also if the value is < 0.95 wouldn't we also want to show 4?

Reply:
ah good catch, wasn't sure if toFixed could be used without parameters, will do thanks!

according to the issue
`With one exception: if the USD price is between $0.95-1.05, then use 4 decimal places.`

https://github.com/Uniswap/interface/issues/2520

Reply:
Okay confirmed with Will that you can use 4 decimals for anything < 0.95.

Reply:
Oh sorry I got it backwards. I mean < 1.05

Reply:
Changed!